lib/Elastica/Aggregation/DateHistogram.php

```diff
@@ -11,7 +12,7 @@ class DateHistogram extends Histogram
     /**
      * Set pre-rounding based on interval.
      *
-     * @deprecated Option "pre_zone" is deprecated as of ES 1.5. Use "time_zone" instead
+     * @deprecated Option "pre_zone" is deprecated as of ES 1.5 and will be removed in Elastica 4.0. Use "time_zone" instead
```

Comment:
Usually deprecated stuff removed at major versions, also it is a good to give developers some directions for future. But it is up to you. I will change it to "and will be removed in further Elastica releases" and make PR for this.